src/app/components/site/Header.tsx

```diff
@@ -4,12 +4,10 @@ import { selectors, useAppSelector } from "../../state";
 import { Col, Container, Row } from "reactstrap";
 import { MainSearch } from "../elements/MainSearch";
 import { NavigationBar } from "./NavigationBar";
-import { useDeviceSize } from "../../services";
 
 export const Header = () => {
   const user = useAppSelector(selectors.user.orNull);
   const mainContentId = useAppSelector(selectors.mainContentId.orDefault);
-  const deviceSize = useDeviceSize();
   return (
     <header className="light" data-testid={"header"}>
```

Comment:
one more from me sorry man, just finesse really. Can we tweak the padding on the login/signup just a little bit more on tablet / desktop. Especially the spacing with the search bar: 

Current: 
<img width="1285" alt="image" src="https://github.com/user-attachments/assets/0cec41a8-d918-40e8-a8e4-fa0ef4ed5147">

Design:
<img width="848" alt="image" src="https://github.com/user-attachments/assets/221f7233-686a-4aca-a2da-23b5ffd33abe">

Reply:
Removed the spacing which was making it come closer to the search bar

![Screenshot 2024-10-17 at 10 49 23](https://github.com/user-attachments/assets/9e506de7-f96b-42d6-9dcb-e344d202b6fb)